lutris/runners/steam.py

```diff
@@ -235,6 +239,14 @@ def get_steamapps_dirs(self):
                     dirs.append(path)
                 i += 1
 
+        # New Custom dirs
+        if library_config := self.get_library_config():
+            paths = [system.fix_path_case(entry["path"] + "/steamapps")
+                     for entry in library_config.values() if entry.get("path")]
```

Comment:
Handle the `mounted` field, supporting both the current stable and beta Steam.
```suggestion
            paths = [system.fix_path_case(entry["path"] + "/steamapps")
                     for entry in library_config.values() 
                     if entry.get("path") and entry.get("mounted") in ("1", None)]
```